pkg/cloud/cloud.go

```diff
@@ -746,6 +752,43 @@ func (c *cloud) batchDescribeVolumesModifications(request *ec2.DescribeVolumesMo
 	return r.Result, nil
 }
 
+// ModifyTags adds, updates, and deletes tags for the specified EBS volume.
+func (c *cloud) ModifyTags(ctx context.Context, volumeID string, tagsToAdd map[string]string, tagsToDelete []string) error {
```

Comment:
`ModifyTags` should probably take a `ModifyTagOptions` instead of the maps separately